Fastfile

```diff
@@ -74,6 +92,7 @@ default_platform :ios
   end
 
   after_all do |lane|
+    notifiy_slack()
```

Comment:
Hmm yeah I see the issue. I guess`after_all` does make the most sense. 

This still gets called on an error, right? The reason I ask is because you can also catch errors in Fastlane and then Slack the exception description. If we did change it so that we don't use `after_all` we could use something like:
```ruby
error do |lane, exception|
  slack(message: "#{exception}")
end
```

The upside to this is you get a clearer understanding of what failed in Slack I think?